block-report.js

```diff
@@ -99,7 +56,7 @@ async function runReport(project, site, indexUrl, cached = true) {
             totals.blocks[blockName] = (totals.blocks[blockName] || 0) + 1;
 
             const links = getNodesByType(table.table, 'link');
-            const name = entry + ',' + blockName + j;
+            const name = entry + ',' + blockName + ',' + j;
```

Comment:
Nit: template string
```suggestion
            const name = `${entry},${blockName},${j}`;
```